modules/apps/27-interchain-accounts/types/account.go

```diff
@@ -21,9 +21,14 @@ func GenerateAddress(moduleAccAddr sdk.AccAddress, portID string) sdk.AccAddress
 	return sdk.AccAddress(sdkaddress.Derive(moduleAccAddr, []byte(portID)))
 }
 
-// ParseAddressFromVersion trims the interchainaccounts version prefix and returns the associated account address
+// ParseAddressFromVersion attempts to extract the associated account address from the provided version string
 func ParseAddressFromVersion(version string) string {
```

Comment:
should we put parsing functions in a separate file? Host/Controller connection parsing doesn't have much to do with `account.go`

`keys.go` is typically where I might put it, open to other ideas

Reply:
Yes, definitely! I was actually going to suggest doing this in a follow up PR next, but I can do it in this. 
I was considering suggesting we that we move the `GeneratePortID` and parsing the conn sequence from port id funcs to `port.go`, and moving the `GenerateAddress` and `ParseAddressFromVersion` to `address.go`, or alternatively we leave those ones here and just move the former to `keys.go`!

Reply:
I like the change you suggested :+1: but we can do this in a followup if we desire. Using `keys.go` works for me. No strong preference